libs/ledgerjs/packages/cryptoassets/src/types.ts

```diff
@@ -1,9 +1,3 @@
-import { cryptocurrenciesById } from "./currencies";
-
-// FIXME This should be the other way around, CryptoCurrencyIds should be defined on its own and cryptocurrenciesById should be derived from it.
-// That way CryptoCurrencyIds could be moved to the types-cryptoassets lib where it belongs.
-export type CryptoCurrencyIds = keyof typeof cryptocurrenciesById;
-
 export type ERC20Token = [
   string,
   string,
```

Comment:
Expect conflicts with upcoming https://github.com/LedgerHQ/ledger-live/pull/1778 unfortunately. For instance with both PRs we'll be able to completely remove this file.